test_ulid2.py

```diff
@@ -20,14 +20,22 @@ def test_ulid_time_monotonic(generator):
         '2013-12-01 10:10:10',
         '2016-07-07 14:12:10',
         '2016-07-07 14:13:10',
+        '2016-07-07 14:13:10',
+        '2016-07-07 14:13:10',
+        '2016-07-07 14:13:10',
     ]:
         dt = datetime.datetime.strptime(time, '%Y-%m-%d %H:%M:%S')
-        ulid = generator(dt)
+        ulid = generator(dt, monotonic=True)
```

Comment:
Shouldn't be it False by default? As monotonic functionality is rather odd and unusual?